.gitignore

```diff
@@ -14,10 +14,8 @@ rust/tmp.env
 tmp.env
 .DS_STORE
 
-typescript/*/.env
 typescript/*/node_modules
-typescript/**/tsconfig.tsbuildinfo
-**/**/tsconfig.tsbuildinfo
```

Comment:
why the changes here? Don't really remember the exact situation where `tsconfig.tsbuildinfo` files are created to be fair but I know they shouldn't be committed

Reply:
I just deduplicated a bit, read the full gitignore and you'll see they were already getting covered